diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8aa9e8b..b699492 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,8 +6,8 @@ cmake_minimum_required(VERSION 3.16...3.28)
 
 project(TurboVNC NONE)
 string(TOLOWER ${CMAKE_PROJECT_NAME} CMAKE_PROJECT_NAME_LC)
-set(VERSION 3.3.1)
-set(DOCVERSION 3.3.1)
+set(VERSION 3.3.1.mt32)
+set(DOCVERSION 3.3.1.mt32)
 set(COPYRIGHT_YEAR "1999-2026")
 set(COPYRIGHT "The VirtualGL Project and many others (see README.md)")
 set(URLTEXT "Visit http://www.TurboVNC.org for more information on TurboVNC")
